README.md

```diff
@@ -1 +1,8 @@
-# TestPRRepo
\ No newline at end of file
+# TestPRRepo
+# PR Rules
```

Comment:
May I request that this be replaced with ## PR Rules to create a subsection

Reply:
Added ## to create the sub section.